lib/sidekiq_unique_jobs/scripts.rb

```diff
@@ -7,35 +7,44 @@
 module SidekiqUniqueJobs
   module Scripts
     LUA_PATHNAME ||= Pathname.new(__FILE__).dirname.join('../../redis').freeze
-    SOURCE_FILES ||= Dir[LUA_PATHNAME.join('**/*.lua')].compact.freeze
-    DEFINED_METHODS ||= [].freeze
     SCRIPT_SHAS ||= Concurrent::Map.new
 
-    module_function
+    include SidekiqUniqueJobs::Connection
 
-    extend SingleForwardable
-    def_delegators :SidekiqUniqueJobs, :connection, :logger
+    module_function
 
     def call(file_name, redis_pool, options = {})
-      internal_call(file_name, redis_pool, options)
+      execute_script(file_name, redis_pool, options)
```

Comment:
Thank you ! it looks like this is fixing the deprecation warning about redis-namespace